AutonomousAI/chat.py

```diff
@@ -46,9 +48,16 @@ def chat_with_ai(prompt, user_input, full_message_history, permanent_memory, tok
         print("----------- END OF CONTEXT ----------------")
 
     response = openai.ChatCompletion.create(
-        model="gpt-4",
+        model="gpt-3.5-turbo",
```

Comment:
Should this be based on if the -gpt3only flag is set? (i.e. set the model to "gpt-4" or "gpt-3.5-turbo" based on that user flag)?

Reply:
Works both ways,